pkg/kbagent/setup.go

```diff
@@ -106,3 +183,80 @@ func deserializeActionNProbe(da, dp string) ([]proto.Action, []proto.Probe, erro
 	}
 	return actions, probes, nil
 }
+
+func runAsServer(logger logr.Logger, config server.Config, services []service.Service) error {
```

Comment:
It may be necessary to check if two ports are duplicated in `config`

Reply:
Done